### ISSUE TITLE
Improve the automatically generated email when placed on the waiting list

### ISSUE THREAD

**Issue by KitClement** [thewca/worldcubeassociation.org#6822]
The current English email for being placed on the waiting list reads as follows:

> Subject: Your registration for [competition] is submitted and pending approval
>
> Your registration has been received and is pending approval from the organization team.
> 
> You will be emailed when your registration is approved. Please note that registrations are not automatically approved, so be patient if you have not been accepted straight away. There may be more registration steps to follow before your registration will be approved. To see the approval criteria visit the competition website or contact the organizers. You can also check the status of your registration here.

This email makes sense when the competition has open space and you are just waiting for the organization team to approve the registration, but it can give a false impression that your registration is merely awaiting approval when the competition is full already.

I would suggest modifying it to the following for the English version:

> Subject: Your registration for [competition] is submitted and pending.
>
> Your registration has been received and is pending approval from the organization team.
> 
> You will be emailed if your registration is approved. Please note that registrations are not automatically approved, so be patient if you have not been accepted straight away. If the competition has already filled, your registration will be placed on a waiting list and will only be accepted if other competitors cancel their registrations. If you are unsure of the status of your registration, please contact the organizers.
> 
> Additionally, there may be more registration steps to follow before your registration will be approved (for example, completing payment). To see the approval criteria visit the competition website or contact the organizers. You can also check the status of your registration here.

**Comment by EdHollingdale** [comment]
I agree that the wording could be improved - although I think there are still some regions which don't use a waiting list, so to handle translation etc it will have to be a bit more general I suspect. Although based on the number of emails we receive a shocking number of people do not read these automated emails anyway.

If we are changing the wording of this email I'd also change the second last sentence to add "(for example, completing payment)" since this is the most common extra step. 

I think a better solution would be to actually split the registrations process into 4 different sections (i.e. pending, waiting list, approved, deleted). I think you Kit already have an issue suggesting that from 2017 or something.

**Comment by EdHollingdale** [comment]
> I wasn't aware that waiting lists were not universally used.
I was mistaken, on doing some more digging it looks like since the WCRP change in 2020 all competitions have waiting lists.

I agree this wording change is a help for those who read them and it would reduce some areas of confusion (e.g. the emails like "can I please be added to the waiting list" etc). But based on many of the other emails I receive it is evident they have not read the email or registration requirements (e.g. thinking their registration was approved based on this automated email despite it quite clearly not saying that, not being aware they need to make payment).

**Comment by dunkOnIT** [comment]
Closed in favour of #10503
